src/main/java/com/minecolonies/commands/ListCitizens.java

```diff
@@ -0,0 +1,180 @@
+package com.minecolonies.commands;
+
+import com.minecolonies.colony.CitizenData;
+import com.minecolonies.colony.Colony;
+import com.minecolonies.colony.ColonyManager;
+import com.minecolonies.colony.IColony;
+import net.minecraft.command.CommandException;
+import net.minecraft.command.ICommandSender;
+import net.minecraft.server.MinecraftServer;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.text.ITextComponent;
+import net.minecraft.util.text.Style;
+import net.minecraft.util.text.TextComponentString;
+import net.minecraft.util.text.TextFormatting;
+import net.minecraft.util.text.event.ClickEvent;
+import org.jetbrains.annotations.NotNull;
+
+import javax.annotation.Nullable;
+import java.util.*;
+
+/**
+ * List all colonies.
+ */
+public class ListCitizens extends AbstractSingleCommand
+{
+
+    /**
+     * Static final string used to draw.
+     */
+    private static final String ID_TEXT                = "§2ID: §f";
+    private static final String NAME_TEXT              = "§2 Name: §f";
+    private static final String COORDINATES_TEXT       = "§2Coordinates: §f";
+    private static final String COORDINATES_XYZ = "§4x=§f%s §4y=§f%s §4z=§f%s";
+    private static final String LIST_COMMAND_SUGGESTED  = "/mc citizens list ";
+    private static final String PAGE_TOP_LEFT = "§2   ------------------ page ";
+    private static final String PAGE_TOP_RIGHT = " ------------------";
+    private static final String PAGE_TOP_MIDDLE = " of ";
+    private static final String PREV_PAGE = " <- prev";
+    private static final String NEXT_PAGE = "next -> ";
+    private static final String PAGE_LINE = "§2 ----------------";
+    private static final String PAGE_LINE_DIVIDER = "§2 | ";
+    private static final int    CITIZENS_ON_PAGE        = 9;
+    private static final String NO_COLONY_FOUND_MESSAGE = "No colony found for id: %d.";
+
+    /**
+     * Initialize this SubCommand with it's parents.
+     *
+     * @param parents an array of all the parents.
+     */
+    public ListCitizens(@NotNull final String... parents)
+    {
+        super(parents);
+    }
+
+    @NotNull
+    @Override
+    public String getCommandUsage(@NotNull final ICommandSender sender)
+    {
+        return super.getCommandUsage(sender) + "";
```

Comment:
That was a placeholder @Raycoms copied that meant "Add your own usage here pelase"